<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Contributors</class>
 <widget class="QWidget" name="Contributors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contributors</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="5">
      <widget class="QLabel" name="lbl_title">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Contributors</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_chzzk">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>[ Chzzk ]</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="lbl_youtube">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>[ Youtube ]</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="3" rowspan="3">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1" rowspan="3">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="5">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="lbl_git">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>[ Git ]</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Tyndall log</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
